src/Tizen.Applications.ComponentManager/Tizen.Applications/ComponentManager.cs

```diff
@@ -0,0 +1,186 @@
+        /// <exception cref="ArgumentException">Thrown when failed because of an invalid argument.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when failed because of the "component not exist" error or the system error.</exception>
+        /// <exception cref="OutOfMemoryException">Thrown when failed because of ouf of memory.</exception>
+        /// <exception cref="UnauthorizedAccessException">Thrown when failed because of permission denied.</exception>
+        /// <privilege>http://tizen.org/privilege/packagemanager.info</privilege>
+        /// <since_tizen> 6 </since_tizen>
+        public static async Task<IEnumerable<ComponentRunningContext>> GetRunningComponentsAsync()
+        {
+            return await Task.Run(() =>
+            {
+                Interop.ComponentManager.ErrorCode err = Interop.ComponentManager.ErrorCode.None;
+                List<ComponentRunningContext> result = new List<ComponentRunningContext>();
+
+                Interop.ComponentManager.ComponentManagerComponentContextCallback cb = (IntPtr contextHandle, IntPtr userData) =>
+                {
+                    if (contextHandle != IntPtr.Zero)
+                    {
+                        IntPtr clonedHandle = IntPtr.Zero;
+                        err = Interop.ComponentManager.ComponentContextClone(out clonedHandle, contextHandle);
+                        if (err != Interop.ComponentManager.ErrorCode.None)
+                        {
+                            Log.Warn(LogTag, "Failed to clone the ComponentInfo. err = " + err);
+                            return false;
+                        }
+                        ComponentRunningContext context = new ComponentRunningContext(clonedHandle);
+                        result.Add(context);
+                        return true;
+                    }
+                    return false;
+                };
+                err = Interop.ComponentManager.ComponentManagerForeachComponentContext(cb, IntPtr.Zero);
+                if (err != Interop.ComponentManager.ErrorCode.None)
+                {
+                    throw ComponentManagerErrorFactory.GetException(err, "Failed to retrieve the running component context.");
+                }
+                return result;
+            }).ConfigureAwait(false);
+        }
+
+        /// <summary>
+        /// Checks whether the component is running or not.
+        /// </summary>
+        /// <param name="componentId">Component ID.</param>
+        /// <returns>Returns true if the component is running, otherwise false.</returns>
+        /// <exception cref="ArgumentException">Thrown when failed because of an invalid argument.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when failed because of the "component not exist" error or the system error.</exception>
+        /// <exception cref="OutOfMemoryException">Thrown when failed because of ouf of memory.</exception>
```

Comment:
Spelling, should be `out` instead of `ouf`.

Reply:
Done